lib/cocoapods/generate/configuration.rb

```diff
@@ -316,6 +316,19 @@ def use_frameworks?
         !use_libraries?
       end
 
+      # @return [String] The project name to use for generating this workspace.
+      #
+      # @param [Specification] spec
+      #        the specification to generate project name for.
+      #
+      def project_name_for_spec(spec)
+        project_name = spec.name.dup
+        # When using multiple Xcode project the project name will collide with the actual .xcodeproj meant for the pod
+        # that we are generating the workspace for.
+        project_name << 'Sample' if generate_multiple_pod_projects?
```

Comment:
can we make this somehow related to `spec.name` so it's visually related? `"#{spec.name}-Gen"` or something similar

Reply:
it already includes `spec.name` we just append 'Sample' for multi-xcodeproj.

Reply:
do you mean just the suffix to append to be different?

Reply:
🤦‍♂ totally misread this, thanks. missed the `<<`